### DISCUSSION TITLE
窗口大小不能调整，不能全屏或者最大化

### DISCUSSION THREAD

**Discussion by JasonWenTheFox** [shion-app/shion#30] [Q&A]
[问题]窗口大小不能调整，不能全屏或者最大化
[使用系统]win11稳定版6月更新
[硬件]5800H 3060lp 两块2k屏幕一块1080p

**Comment by hanaTsuk1** [comment]
刻意限制的，调整大小就要调整布局，耗费时间又对功能没有提升，以后会弄
